__tests__/articles.test.js

```diff
@@ -122,3 +121,88 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("201: returns the successfully posted comment", () => {
+    const testComment = {
+      comment: {
+        username: "butter_bridge",
+        body: "Bemused of Basingstoke. What is this guy on?",
+      },
+    };
```

Comment:
Can we add another 201 test here please? This should ignore unnecessary properties, i.e. if something other than username and body is sent.